docs/TokenCredentialAuthenticationProvider.md

```diff
@@ -54,3 +54,19 @@ const client = Client.initWithMiddleware({
 });
 const res = await client.api("/users/").get();
 ```
+
+The `TokenCredentialAuthenticationProvider` and the `TokenCredentialAuthenticationProviderOptions` are bundled into the `lib/graph-client-tokenCredentialAuthProvider.js` file in an `iife` format.
+
+The browser use of the file is as follows:
+
+```javascript
+// include the script
+<script type="text/javascript" src="<PATH_TO_SCRIPT>/graph-client-tokenCredentialAuthProvider.js"></script>;
+
```

Comment:
suggestion box doesn't allow me to escape three backticks because suggestion itself is using markdown code block.

What I am trying to propose here is that please introduce one code block for html and use html comments for `include the script` line and another code block that for javascript. It would be easier that way for readers of this documentation to copy paste the code into their existing html/javascript.

```suggestion
```html
<!-- include the script -->
<script type="text/javascript" src="<PATH_TO_SCRIPT>/graph-client-tokenCredentialAuthProvider.js"></script>;
```